### DISCUSSION TITLE
KEDA Chart: v2.16.0

### DISCUSSION THREAD

**Discussion by JorTurFer** [kedacore/charts#705] [Announcements]
Chart: v2.16.0 | App: v2.16.0

## TL;DR

```shell
helm repo add kedacore https://kedacore.github.io/charts
helm repo update
helm install keda kedacore/keda --version 2.16.0 -n keda --create-namespace
```

### Fixes/Changes
- Support for KEDA v2.16.0
- Replace wildcards in RBAC objects with explicit resources and verbs (https://github.com/kedacore/charts/pull/682
- Update cloudeventsource type list (https://github.com/kedacore/charts/pull/683)
- Add appProtocol to services collecting metrics using mTLS (https://github.com/kedacore/charts/pull/688)
- Update status about triggers and authentications types (https://github.com/kedacore/charts/pull/700)

## New Contributors
* @seankhliao made their first contribution in https://github.com/kedacore/charts/pull/678
* @noyoshi made their first contribution in https://github.com/kedacore/charts/pull/684
* @Rotzke made their first contribution in https://github.com/kedacore/charts/pull/674
* @legal90 made their first contribution in https://github.com/kedacore/charts/pull/682
* @guicholeo made their first contribution in https://github.com/kedacore/charts/pull/688
* @or-shachar made their first contribution in https://github.com/kedacore/charts/pull/694

**Full Changelog**: https://github.com/kedacore/charts/compare/v2.15.1...v2.16.0

<hr /><em>This discussion was created from the release <a href='https://github.com/kedacore/charts/releases/tag/v2.16.0'>KEDA Chart: v2.16.0</a>.</em>
